src/attr/_make.py

```diff
@@ -1873,64 +1874,136 @@ def _add_eq(cls, attrs=None):
 
 _already_repring = threading.local()
 
+if HAS_F_STRINGS:
+
+    def _make_repr(attrs, ns, cls):
+        unique_filename = "repr"
+        # Figure out which attributes to include, and which function to use to
+        # format them. The a.repr value can be either bool or a custom
+        # callable.
+        attr_names_with_reprs = tuple(
+            (a.name, repr if a.repr is True else a.repr, a.init)
```

Comment:
```suggestion
            (a.name, (repr if a.repr is True else a.repr), a.init)
```

This took me *minutes* to decipher. 🙈